### ISSUE TITLE
Implement accessibilityActions

### ISSUE THREAD

**Issue by jonthysell** [microsoft/react-native-windows#2765]
Right now, the only action a UIA client can make to a RNW app is Invoke and Toggle (which calls Invoke). The remaining actions we expose to UIA are no-ops:

* [ ] Select
* [ ] AddToSelection
* [ ] RemoveFromSelection
* [ ] Expand
* [ ] Collapse
* [ ] Toggle (re-do)

RN60 adds an accessibilityActions property, which lets you register an event handler to catch these invocations from UIA poking the native side. See docs here: https://facebook.github.io/react-native/docs/accessibility#accessibility-actions

Since it just uses string identifiers, this would be as simple as:

* [ ] Storing the registered `accessibilityActions` event names via `DynamicAutomationProperties`
* [ ] Storing a new `AccessibilityActionsEventHandler` property similar to `AccessibilityInvokeEventHandler` but with a name parameter
* [ ] Having each UIA action list above check the registered names and if specific ones exist ("toggle", "select", etc) then fire the `AccessibilityActionsEventHandler`

**Comment by harinikmsft** [comment]
@jonthysell  - please sync with @acoates-ms  who is driving the upgrade to RN 0.60 and you can start making these changes in the branch where the upgrade is happening in order to avoid any throw away/conflicting work.

**Comment by chrisglein** [comment]
With the RN60 upgrade almost wrapped up this should be ready for attention. @licanhua I'm adding it to your queue alongside another block of accessibility work.

**Comment by licanhua** [comment]
@jonthysell , please help to revisit your proposal. I didn't see any value right now to implement accessibility-actions. Currently dynamicAutomationPeer is applied to View, and adding Toggle, expand action doesn't make any sense.
